joomla/joomla-cms
# [4.2] [Smart Search] Moved statistics in the toolbar

Pull Request for Issue # .

### Summary of Changes
The statistics button is right in the middle between the actions, which I found a little illogical. I suggest moving it to the very end of the toolbar.

![Screenshot_1](https://user-images.githubusercontent.com/8440661/182330765-97c95be4-d972-48f5-becf-9bcb37aba814.png)

### Testing Instructions
Pay attention to the screenshot, after applying the PR, check pages **Smart Search: Indexed Content**, **Smart Search: Content Maps**, **Smart Search: Search Filters**.
